### PR TITLE
Split base boxes into separate files

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: boxes.d/01-buitin.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+boxes:
```
Comment (ehelms) [inline boxes.d/01-buitin.yaml]: Typo on the filename?

Comment (ekohl) [reply boxes.d/01-buitin.yaml]: Fixed

**Comment by ehelms** [comment]
Looks like a failure here in the tests

**Comment by ehelms** [comment]
This needs a rebase

**Comment by ekohl** [comment]
Rebased

File: boxes.d/01-builtin.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    memory: 8096
+    ansible:
+      playbook:
+        - 'playbooks/katello.yml'
+        - 'playbooks/katello_provisioning.yml'
+      group: 'server'
+
+  runcible:
+    box: centos7
+    shell: '/vagrant/shells/runcible.sh'
```
Comment (ehelms) [inline boxes.d/01-builtin.yaml]: Looks like this will need an update with my recent merge.

Comment (ekohl) [reply boxes.d/01-builtin.yaml]: Keep moving targets ....

**Comment by ekohl** [comment]
Once again, updated :)

**Pull request merged**